pkg_resources/__init__.py

```diff
@@ -3464,7 +3468,10 @@ def __contains__(self, item: Distribution | str | tuple[str, ...]) -> bool:
         # Allow prereleases always in order to match the previous behavior of
         # this method. In the future this should be smarter and follow PEP 440
         # more accurately.
-        return self.specifier.contains(item, prereleases=True)
+        return self.specifier.contains(
+            item,  # type: ignore[arg-type]
```

Comment:
GitHub doesn't allow me to add suggestions that far back in unchanged lines, but here's what the proper fix to ensure `item` isn't seen as potentially a `Distribution` by mypy.
```python
            _item = item.version
        else:
            _item = item

        # Allow prereleases always in order to match the previous behavior of
        # this method. In the future this should be smarter and follow PEP 440
        # more accurately.
        return self.specifier.contains(
            _item,
```

Reply:
Done in 9eb89def7.